src/refiners/foundationals/latent_diffusion/stable_diffusion_xl/__init__.py

```diff
@@ -11,4 +12,6 @@
     "SDXLAutoencoder",
     "SDXLIPAdapter",
     "SDXLT2IAdapter",
+    "ControlLora",
```

Comment:
Note (and FYI): I just realize we do not import (yet) `Controlnet` in `src/refiners/foundationals/latent_diffusion/stable_diffusion_1/__init__.py` (it has no docstring yet, though)

Reply:
Noted!